app/assets/javascripts/tree.js

```diff
@@ -83,8 +70,10 @@ Tree.prototype.init = function () {
   }
 
   findTreeitems(this.domNode, this, false);
-
   this.updateVisibleTreeitems();
+  this.updateFileCount();
+  this.searchInput = document.getElementById('sul-embed-search-input');
```

Comment:
Please use a data attribute for this selector rather than using an id. This makes it easier to understand that the node is bound to some javascript.

Reply:
Done.